src/js/utils/animate.js

```diff
@@ -1,4 +1,4 @@
-import { getBarHeightAndYAttr } from './draw-utils';
+import { getBarHeightAndYAttr, createSplineCurve } from './draw-utils';
```

Comment:
Perhaps we can rename the function to `getSplineCurvePointsStr` as the word `create` sounds closer to the actual drawing (rendering) of the graph and does not seem a good name for a utility function

```suggestion
import { getBarHeightAndYAttr, getSplineCurvePointsStr } from './draw-utils';
```